<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeLink</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/images/app_theme.png</normaloff>:/images/images/app_theme.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget_usrlst">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>231</width>
      <height>621</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="icon">
      <iconset resource="src.qrc">
       <normaloff>:/images/images/client_ico.png</normaloff>:/images/images/client_ico.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ip地址</string>
     </property>
     <property name="icon">
      <iconset resource="src.qrc">
       <normaloff>:/images/images/ip_ico.png</normaloff>:/images/images/ip_ico.png</iconset>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_head">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="src.qrc">:/images/images/administor.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Usrname">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>171</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Usrname</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_chatzone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_inputzone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>751</width>
      <height>121</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>640</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>快捷键Ctrl+Enter</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="icon">
     <iconset resource="src.qrc">
      <normaloff>:/images/images/send_ico.png</normaloff>:/images/images/send_ico.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+Return</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>510</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_filesnd">
       <property name="toolTip">
        <string>发送文件</string>
       </property>
       <property name="icon">
        <iconset resource="src.qrc">
         <normaloff>:/images/images/file_ico.png</normaloff>:/images/images/file_ico.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_chatsavr">
       <property name="toolTip">
        <string>保存聊天记录</string>
       </property>
       <property name="icon">
        <iconset resource="src.qrc">
         <normaloff>:/images/images/save_ico.png</normaloff>:/images/images/save_ico.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_p2pchat">
       <property name="toolTip">
        <string>开启私聊</string>
       </property>
       <property name="icon">
        <iconset resource="src.qrc">
         <normaloff>:/images/images/message_ico.png</normaloff>:/images/images/message_ico.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>状态</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Log_out"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_Log_out">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/images/log_out.png</normaloff>:/images/images/log_out.png</iconset>
   </property>
   <property name="text">
    <string>退出登录(Log out)</string>
   </property>
   <property name="toolTip">
    <string>退出当前账号</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
